src/pages/components/tabs/style.mdx

```diff
@@ -5,6 +5,21 @@ description:
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink> <AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink> <AnchorLink>Size</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
```

Comment:
```suggestion
<AnchorLink>Color</AnchorLink> <AnchorLink>Typography</AnchorLink>
<AnchorLink>Structure</AnchorLink>
```

Reply:
It looks like you don't have a section for size and feedback, so you might want to take out those anchor links (unless you want to add in that content)